decidim-participatory_processes/app/views/decidim/participatory_processes/admin/participatory_processes/_form.html.erb

```diff
@@ -32,11 +32,11 @@
       </div>
 
       <div class="row column">
-        <%= form.translated :editor, :short_description, aria: { label: :short_description } %>
+        <%= form.translated :editor, :short_description, resource_mentionable: true, aria: { label: :short_description } %>
```

Comment:
This is not working as it should. I can mention the resource, but in  the frontend the mention si not linked. 
![image](https://github.com/user-attachments/assets/5d486571-3387-428c-835c-2403aac5165b)

Reply:
@alecslupu, fixed now. I thought most of the rich text fields had the processors in place but I guess only those that already support mentions or hashtags do. You'll have to do the mentions again because the once you created were not processed correctly.

Reply:
oh wait, looking at the failing tests, I'll have to do a fix.